libs/ui/src/reports/layout.tsx

```diff
@@ -0,0 +1,43 @@
+import styled, { DefaultTheme } from 'styled-components';
+import { makeTheme } from '../themes/make_theme';
+
+export const TallyReportColumns = styled.div`
+  columns: 3;
+  column-gap: 0.3in;
+  margin-top: 1.5em;
+
+  & > div {
+    margin-top: 0;
+  }
+`;
+
+export const PrintedReport = styled.section`
+  font-size: 12px;
+  page-break-after: always;
+
+  @media print {
```

Comment:
I think that both do?

A bunch or report components are defined in `em` so setting the base font size of the report does affect what is rendered.

The page break is important because otherwise, in the few cases where we're rendering multiple reports into one document, reports always start on a new page. For example, for a primary tally report, we wouldn't want the Fish Party Tally Report header and report to start at the bottom of the Mammal Party Tally Report page. Without this style, I think that would happen.

Reply:
Ah but, for this particular report, I'm not sure if it does anything. But I can test with snapshots to see.

Reply:
looks like I'm using `em` in the metadata so will keep something like this wrapper for now

Reply:
ah, I was referring to just the `@media print {}` section -- was wondering if the PDF is rendered within a `@print` context or `@screen` and whether the `@print` styles have any effect. wasn't clear why we have to declare `font-size` twice

Reply:
it's rendered in a print context. now that all reports are rendered in a print context, the non-print styling could be removed. but while some other things are still printing on the frontend and the transition is not complete, i'll leave the double styling